doc/reference/networking/thread.rst

```diff
@@ -40,3 +40,8 @@ which provide out-of-the-box configuration for OpenThread. To enable OpenThread
 support in these samples, build them with ``overlay-ot.conf`` overlay config file.
 See :ref:`sockets-echo-server-sample` and :ref:`sockets-echo-client-sample` for
 details.
+
+Integration with Zephyr
```

Comment:
Maybe `L2 integration` would be better? After all, the whole page speaks of integrating Thread with Zephyr (as in line 23).